people/luan/index.html

```diff
@@ -12,17 +12,43 @@
             <div>
                 <h2 id="name">Luan's Demo Page</h2>
                 <span id="content">
-                Press cmd + shift + j to open developer tools
+                <h1></h1>
                 </span>
             </div>
             <br>
         </div>
 
+        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
         <script type="text/javascript">
-          var name  = prompt("Enter your name");
-          var title = prompt("Enter your title");
-          
-          console.log(name + " is a " + title + " at Invoca");
+            var today = new Date();
+            var time = today.getHours();
+
+            var name  = prompt("Enter your name");
+
+            function capitalizeFirstLetter(string) {
+                return string.charAt(0).toUpperCase() + string.slice(1);
+            };
+
+            function checkTime() {
+                if (time > 0 && time <= 12) {
+                    $("h1").append("Good Morning, " + capitalizeFirstLetter(name));
+                    $("body").css("background-image", "url('morning.jpg')");
+                    $(".invoca img").css("margin", "200px 0 0 0")
+                } else if (time > 12 && time < 18) {
+                    $("h1").append("Good Afternoon, " + capitalizeFirstLetter(name))
+                    $("body").css("background-image", "url('afternoon.jpg')");
+                } else {
+                    $("h1").append("Good Evening, " + capitalizeFirstLetter(name));
+                    $("body").css("background-image", "url('night1.jpg')");
+                    $("h1").css("color", "#ffffff")
+                }
+            };
+
+            checkTime();
+            
+           if (name == "Garvan" || "garvan") 
```

Comment:
`if (name == "Garvan" || "garvan") ` will always evaluate to being truthy... so no matter who's name is entered the alert will always trigger.

You are trying to check if `name == "Garvan"` or if `name == "garvan"`. 
`if (name == "Garvan" || "garvan")` will evaluate to true because even if `name` does **not** equal "Garvan" it will evaluate `"garvan"` as being true.